### PR TITLE
[fixed] missing commas, reformatted code more easily readable code and change to fstring format

### PR DESCRIPTION
**Pull request by slowy07**
fixing on ```experiment/run_experiment.py``` contains:
- [x] adding missing commas
- [x] reformatted code for more easily readable
- [x] and change ```%s``` format to ```fstring``` format for more readable and less of errors but fast  
- [x] passed tested with flake8 
- [x] reformatted code without change str quote marks

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
I appreciate this change but it changes all of our single quotes to double quotes. We don't want that.

**Comment by slowy07** [comment]
> I appreciate this change but it changes all of our single quotes to double quotes. We don't want that.

okay, solved @jonathanmetzman ! 👍🏻

File: experiment/run_experiment.py [hunk 1]
```diff
@@ -88,28 +88,27 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
     for param in required_params:
         if param not in config:
             valid = False
-            logs.error('Config does not contain "%s".', param)
+            logs.error(f'Config does not contain "{str(param)}".')
```
Comment (jonathanmetzman) [inline experiment/run_experiment.py:91]: This pattern is going to beak presubmit.
The standard pattern we adhere to is to use % format strings for logs, not f-strings.

File: experiment/run_experiment.py [hunk 2]
```diff
@@ -402,15 +405,16 @@ def start(self):
...
             '/bin/bash',
             '-c',
-            'rsync -r '
-            '"${EXPERIMENT_FILESTORE}/${EXPERIMENT}/input/" ${WORK} && '
-            'mkdir ${WORK}/src && '
-            'tar -xvzf ${WORK}/src.tar.gz -C ${WORK}/src && '
-            'PYTHONPATH=${WORK}/src python3 '
-            '${WORK}/src/experiment/dispatcher.py || '
-            '/bin/bash'  # Open shell if experiment fails.
+            'rsync -r ',
```
Comment (jonathanmetzman) [inline experiment/run_experiment.py:408]: I don't understand this change. This is supposed to be one argument and you made it many.

**Review by jonathanmetzman — COMMENTED** [review]
Sorry i don't think we want this. I didn't realize what it's purpose was but it mostly is bad (changing % format strings in logging to f-strings. Thank you for your help, but I don't think we will land this.

**Pull request closed without merging**